lib/fathead/lodash/parse.js

```diff
@@ -0,0 +1,46 @@
+"use strict";
+
+var cheerio = require("cheerio");
+var fs = require("fs");
+var formattedData = [];
+
+fs.readFile("./download/lodash.html", "utf-8", function (err, lodashDoc) {
+    if (err) throw err;
+    var $ = cheerio.load(lodashDoc);
+    var $docContainer = $(".doc-container"); //This contains all the docs
+
+    var _loop = function _loop(i) {
+        var $currElem = $docContainer.children("div").eq(i); //current children of the doc
+        $currElem.children("div").map(function (index) {
+            return formatDataAndPush($currElem.children("div").eq(index));
+        }); //Every children in the current children which is a method
+    };
+
+    for (var i = 0; i < $docContainer.children("div").length - 2; i += 1) {
+        _loop(i);
+    }
+    fs.writeFileSync("./output.txt", formattedData.join(''));
+});
+
+function formatDataAndPush($elem) {
+    var _getDetails = getDetails($elem),
+        article = _getDetails.article,
+        url = _getDetails.url,
+        abstract = _getDetails.abstract;
+
+    formattedData.push("._" + article + "()\tA\t\t\t\t\t\t\t\t\thttps://www.drupal.org/files/project-images/logo%20(2).png\t" + abstract + "\t" + url + "\n", article + "\tR\t._" + article + "()\t\t\t\t\t\t\t\t\t\t\n", "._" + article + "\tR\t._" + article + "()\t\t\t\t\t\t\t\t\t\t\n");
+}
+
+function getDetails($elem) {
+    var article = $elem.children("h3").attr("id");
+    var url = "https://lodash.com/docs/4.17.4#" + article;
+    var abstractData = {
+        title: $elem.children("h3").children("code"),
+        summary: $elem.children("p").eq(1).text(),
+        args: $elem.children("ol"),
+        returns: $elem.children("p").eq(3),
+        example: $elem.children(".highlight").html()
```

Comment:
Until you remove the `<div>` tags, this won't be able to upload to beta, as our processing code tosses out abstracts containing `<div>`